cfp.md

```diff
@@ -1,9 +1,64 @@
+
+The BSides community has continuously raised the bar and put the info back in infosec. We thank each and every member/participant/organizer of this community for their hard work, sweat, and relentless pursuit of making high quality content accessible. We celebrate diverse approaches and points of view and believe that our commitment to diversity fosters innovation. BSidesSF is continuing to build upon a culture that drives our community to include a wide range of thoughts and ideas.
+
+**When:**  BSidesSF will take place in San Francisco from February 22-24, 2020.  The Call for Participation for talks and workshops will close on November 6, 2019 at 11:59 PM PST.
+
+**How** 
+This year, we are expanding the type of submissions we are supporting at BSidesSF. New session types include deep dive talks, lightning talks and an off the record track. 
+
+**Workshops: February 22, 2020** 
+Please note that workshops will be held on Saturday, February 22, 2020 at a different location.
+
+- **Condensed workshops** - 2 hr 45 mins
+- **Full length workshops** - 5 hr 30 mins 
+
+**Talks: February 23-24, 2020**
+Please note that all talks are limited to two speakers only. Panels are limited to 5 participants (including the moderator).
+ 
+- **General talks** - 25 min regular talk format for various topics in security and privacy.
+- **Deep dive talks** - 50 min talk format to cover more technical deep dives.
+- **Panels** -  50 min panel discussion to bring together a group of experts on a broadly accessible topic.
+- **Lightning talks** - 10 min lightning talk about a new idea or project that you are exploring.
+- **Off the record** - 25 min or 50 min talks for topics that you would like to discuss “Off the record”. We will be enforcing no recording/photography for this track.   
+
+**Important Dates**
+- September 25, 2019 – Call for submissions opens
+- October 16, 2019 – Papers and Workshop proposals for early-bird consideration due
+- October 30, 2019 – First round speakers and trainers notified
+- November 6, 2019 – Final due date for all submissions
+- November 24, 2019 – Notifications sent
+- December 4, 2019 - Participation confirmed by speakers and details finalized
+- February 22-24, 2020 – BSidesSF 2020
+
+## What You Need
+- Speaker name(s) and contact information
+- Bio(s) 
+- Submission Title
+- Abstract - Limited to 200 words
+- Detailed Outline and Key Takeaways - Tell us more details about your presentation and why people will want to attend. These details will help us make an informed decision about your submission. 
+- T-shirt size(s) 
+- Are you interested in working with a program committee member for a dry run? 
+- Would you like to donate your speaker gift to one of our charities? 
+- Scheduling preferences 
+
+## To Submit
+
+[Submit a Talk](https://www.papercall.io/bsidessf-2020)
+
+[Submit a Workshop](https://forms.gle/mjgApRdV2S9WATZp8)
```

Comment:
Hmm, not able to use papercall for this as well?

Reply:
No, unfortunately. The Workshops require more customization that we can eek out of Papercall, but also don't need the same level of crazy spreadsheet math either.